src/main/java/core/mate/academy/producers/BulldozerProducer.java

```diff
@@ -0,0 +1,14 @@
+package core.mate.academy.producers;
+
+import core.mate.academy.model.Bulldozer;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer {
+    @Override
+    public List get() {
+        Bulldozer bulldozer1 = new Bulldozer("gt140", "red", "pink", 240);
```

Comment:
As I understood from condition of exercise, MachineProducer and all other producers implementations method get() should return the list of specific machines. For example:
`List<Bulldozer> get()`. I mean, you return simply List, but it should be parameterized.